www-src/app/replicator/fromDevice/event_synchronizer.coffee

```diff
@@ -95,7 +97,15 @@ module.exports = class EventSynchronizer
             _deleted: true
 
         @db.put toDelete, toDelete._id, toDelete._rev, (err, res) =>
-            return callback err if err
-
-            @calendarSync.deleteEvent androidEvent, \
-                    AndroidAccount.ACCOUNT, callback
+            if err
+                if err.status is 409
+                    @conflictsHandler.handleConflicts toDelete, (err, doc) =>
```

Comment:
Not sure it's the right conflit you're trying to solve here : 409 are for "immediate conflicts" (trying to update a doc from a to older version ; but conflictHandler work on eventual conflict ( http://pouchdb.com/guides/conflicts.html )
But right, we have to handle immediate conflict in import code !